src/main/resources/templates/editBookstore.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="UTF-8">
+    <title>Edit Bookstore</title>
+</head>
+<body>
+
+    <!-- Upload Books-->
+    <h2>Upload Book</h2>
+    <form id="addBook" name="addBook" th:action="@{/uploadBook}" method="post" class="container">
+        <label>Title:</label><br>
+        <input type="text" name="title"><br><br>
+        <label>ISBN:</label><br>
+        <input type="text" name="isbn"><br><br>
+        <label>Description:</label><br>
+        <input type="text" name="description"><br><br>
+        <label>Author:</label><br>
+        <input type="text" name="author"><br><br>
+        <label>Publisher:</label><br>
+        <input type="text" name="publisher"><br><br>
+        <label>Genre:</label><br>
+        <input type="text" name="genre"><br><br>
```

Comment:
Currently when you add a book there are 0 of them. When the owner goes to upload a book, should they not also specify the quantity of said book?

Reply:
Thanks for catching that, I will make this update now.